pkg/apis/core/types.go

```diff
@@ -3508,6 +3512,14 @@ type LoadBalancerIngress struct {
 	// (typically AWS load-balancers)
 	// +optional
 	Hostname string
+
+	//The list has one entry per service port in the manifest
+	// +optional
+	PortStatuses []PortStatus
+
+	//A human readable message indicating details about the condition of the load-balancer.
```

Comment:
Is human-readable what we want here?  We have conditions and ports which have messages - what is THIS for that isn't better in one of those?

Reply:
(I just wonder how I should interpret all caps here).
Well, this is the part of the agreed and accepted KEP. Which has been reviewed and accepted by people much better in K8s API design than I.
Of course things change and I am OK to remove it from here. Just like the whole API implementation as said above.

Reply:
/KEPs are not written in stone, and API reviews can tweak the exact structure.  I am asking what this particular field is for that is not already covered by the other messages?